sepia-fog-images/config/definitions/sepia-fog-images.yml

```diff
@@ -32,6 +32,10 @@
           name: CMANSIBLEBRANCH
           default: "master"
           description: "Optionally define a different ceph-cm-ansible branch (useful for testing)"
+      - string:
+          name: PAUSEQUEUE
+          default: "true"
```

Comment:
are you sure the quotes are needed? I wonder if this might become double double-quoted

Reply:
![screenshot at 2019-01-17 09-20-53](https://user-images.githubusercontent.com/11281208/51324633-3bc90a80-1a39-11e9-86cd-65f4444858e1.png)